test/checks/color/color-contrast.js

```diff
@@ -906,6 +906,29 @@ describe('color-contrast', function () {
     }
   );
 
+  (shadowSupported ? it : xit)('handles <slot> elements', () => {
+    fixture.innerHTML =
+      '<div id="container" style="height: 100px;"><p id="target" style="color: #333;">Slotted text</p></div>';
+    const container = fixture.querySelector('#container');
+    const shadow = container.attachShadow({ mode: 'open' });
+
+    shadow.innerHTML =
+      '<div id="shadowContainer" style="position: absolute; background: black;"><slot></slot></div>';
+    const shadowContainer = shadow.querySelector('#shadowContainer');
+    axe.testUtils.flatTreeSetup(fixture);
+
+    const target = fixture.querySelector('#target');
```

Comment:
Couldn't use `axe.utils.querySelector` since the element is in the shadow dom in the virtual tree, so would need to select the shadow element and query selector from there.